draft-ietf-acme-acme.md

```diff
@@ -938,10 +939,12 @@ resulting account object.  This allows clients to detect when servers do not
 support an extension field.
 
 The server SHOULD validate that the contact URLs in the "contact" field are
-valid and supported by the server.  If the client provides the server with an
-invalid or unsupported contact URL, then the server MUST return an error of type
-"invalidContact", with a description describing the error and what types of
-contact URL the server considers acceptable.
+valid and supported by the server. If the client provides the server with
+a contact URL using an unsupported protocol it MUST return an error of type
+"unsupportedContact". The server MUST support the "mailto:" protocol. If the
```

Comment:
> "mailto:" protocol

Based on [RFC 6068](https://tools.ietf.org/html/rfc6068) and [RFC 3986](https://tools.ietf.org/html/rfc3986#section-3.1), it seems like we should refer to this as "mailto scheme" (no quotes, no colon, using scheme instead of protocol).

> The server MUST support the "mailto:" protocol.

I think some context is warranted here; we don't actually expect ACME servers to support mailto (i.e., for the server to implement the mailto protocol), we expect ACME servers to support the mailto scheme _for contact URIs_.